instrumentation/runtime-metrics/runtime-metrics-java8/library/src/main/java/io/opentelemetry/instrumentation/runtimemetrics/java8/Cpu.java

```diff
@@ -89,7 +89,7 @@ List<AutoCloseable> registerObservers(
         meter
             .gaugeBuilder("process.runtime.jvm.system.cpu.load_1m")
             .setDescription("Average CPU load of the whole system for the last minute")
-            .setUnit("1")
+            .setUnit("{core}")
```

Comment:
this is related to https://github.com/open-telemetry/semantic-conventions/pull/95

Reply:
I'm not sure about `{core}`, see https://github.com/open-telemetry/semantic-conventions/pull/95#discussion_r1222045510

Reply:
fixed to "{run_queue_length}" as discussed